python/tests/arithmetic_tests/test_timestamp.py

```diff
@@ -8,105 +8,206 @@ def __init__(self):
         self.sql = """CREATE LOCAL VIEW atimestamp_minus_timestamp AS SELECT
                       id,
                       (c1-c2)SECOND AS seconds,
+                      (c1-c2)MINUTE AS minutes,
```

Comment:
we now should support all intervals, including hours to minutes, etc.
there are 10 types of short intervals and 3 of long intervals.
these can be a separate PR